packages/ui/src/components/molecules/DropDownListItem.tsx

```diff
@@ -185,6 +187,7 @@ export const DropDownListItem: FC<DropDownListItemProps> = ({
             variant="h6"
             $color={color ?? 'muted'}
             $isChecked={checked}
+            $restrictedItem={$restrictedItem}
```

Comment:
The UI still needs some work 
![image](https://github.com/Cypherock/cypherock-cysync/assets/107414753/b5e2df7c-5885-47a3-8fb8-5e6f6ff2a549)